python/pyspark/streaming/kinesis.pyi

```diff
@@ -32,15 +35,15 @@ class KinesisUtils:
         regionName,
         initialPositionInStream,
         checkpointInterval,
-        storageLevel: Any = ...,
-        awsAccessKeyId: Optional[Any] = ...,
-        awsSecretKey: Optional[Any] = ...,
-        decoder: Any = ...,
-        stsAssumeRoleArn: Optional[Any] = ...,
-        stsSessionName: Optional[Any] = ...,
-        stsExternalId: Optional[Any] = ...,
+        storageLevel: StorageLevel = ...,
+        awsAccessKeyId: Optional[str] = ...,
+        awsSecretKey: Optional[str] = ...,
+        decoder: Callable[[T], T] = ...,
```

Comment:
This should be

```python
Callable[[Optional[bytes]], T] = ...,
```

shouldn't it?

And since w already modify the file, maybe 

```python
def utf8_decoder(s: Optional[bytes]) -> str: ...
```

Reply:
Absolutely. Thanks!